contrib/galaxy_supervisor.conf

```diff
@@ -23,8 +23,7 @@ autostart       = true
 autorestart     = true
 startsecs       = 20
 user            = galaxy
-environment     = PATH=/home/galaxy/galaxy/.venv/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin,PYTHONHOME=/home/galaxy/galaxy/.venv
-numprocs        = 1
+environment     = PATH=/home/galaxy/galaxy/.venv/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```

Comment:
We can simplify this to:
`environment     = PATH=/home/galaxy/galaxy/.venv/bin:%(ENV_PATH)s`

Reply:
That sounds good.